build-rpi-raspbian/webif.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+# start:     	starts a minimalistic web server
+# 	    	shows the status of OpenHABianPi installation
+# restart:   	checks if webserver is running 
+# inst_done: 	create finish message and link to http://$hostname:8080
+# cleanup:	stops the webserver
+#	     	removes all no longer needed files
+
+port=80 # Port the webserver is listing to
+source /etc/openhabian.conf # to get the hostname
+
+if [ $1 = "start" ]; then
+  mkdir /tmp/webif
+  ln -s /boot/first-boot.log /tmp/webif/first-boot.txt
+  echo "<html>
+        <head>
+        <title>openHABian</title>
+        <meta http-equiv="refresh" content="5" />
+        <meta http-equiv="Cache-Control" content="no-cache, no-store, must-revalidate" />
+        <meta http-equiv="Pragma" content="no-cache" />
+        <meta http-equiv="Expires" content="0" />
+        </head>" > /tmp/webif/index.html
+  echo "<body>
+        <h1>openHABian Installation Status</h1>
+        the log will be refreshed automatically every 10 seconds
+        <iframe src="http://$hostname:$port/first-boot.txt" scrolling="yes" width="100%" height="90%"></iframe>
+        </body>
+        </html>" >> /tmp/webif/index.html
+  python3 -m http.server $port > /dev/null 2>&1 &
```

Comment:
I'd actually suggest to remove `2>&1` as it's stderr output could certainly be useful.

Reply:
Also the command as it is will not show your content. I'd suggest:

```
(cd /tmp/webif; python3 -m http.server $port > /dev/null)
```

Reply:
yes it removes the stderr ourput, but also the normal output, otherwise we will see the http-request messages every 10 seconds and i found this really annoying

thank you for the command part, i will do this in the next commit